drawOscilloscope.js

```diff
@@ -1,9 +1,9 @@
 function drawOscilloscope(samples, x, y, rectWidth, rectHeight) {
```

Comment:
I originally had it pass in x, y, rectWidth, and rectHeight, but since it seems you could access `height` and `width` within this scope could you remove these arguments?